src/test/kotlin/no/nav/familie/ef/sak/infrastruktur/config/HistoriskPensjonMock.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.familie.ef.sak.no.nav.familie.ef.sak.infrastruktur.config
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.pensjon.HistoriskPensjonClient
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.pensjon.HistoriskPensjonResponse
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+
+class HistoriskPensjonMock {
+
+    @Profile("mock-historiskpensjon")
+    @Bean
+    @Primary
+    fun historiskPensjonClient(): HistoriskPensjonClient {
+        val mockk = mockk<HistoriskPensjonClient>()
+        every { mockk.harPensjon(any()) } returns HistoriskPensjonResponse(false, "")
```

Comment:
Kan være fint å returnere true som default for å vise den i frontend :) med en random lenke